RealmTasks Android/app/src/main/java/io/realm/realmtasks/TaskActivity.java

```diff
@@ -155,20 +154,17 @@ public void onMoved(RecyclerView recyclerView, ItemViewHolder from, ItemViewHold
             final int fromPosition = from.getAdapterPosition();
             final int toPosition = to.getAdapterPosition();
             adapter.onItemMoved(fromPosition, toPosition);
-            adapter.notifyItemMoved(fromPosition, toPosition);
```

Comment:
why were we calling this before?

Reply:
Because we have support for fine-grained notifications before.